instrumentation/opensearch/opensearch-rest-1.0/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.opensearch.client")
+    module.set("opensearch-rest-client")
+    versions.set("[1.0,)")
+    assertInverse.set(true)
+  }
+
+  fail {
+    group.set("org.opensearch.client")
+    module.set("rest")
+    versions.set("(,)")
+  }
+}
+
+otelJava {
+  minJavaVersionSupported.set(JavaVersion.VERSION_11)
+}
+
+dependencies {
+  library("org.opensearch.client:opensearch-rest-client:1.3.6")
```

Comment:
can this test against 1.0 version? and the testLatestDeps build will automatically test against the latest

Reply:
This is not a test dependency, so using an older version that has been patched multiple times doesn't look like a good idea.
Did you mean the testcontainers' OpenSearch version should be 1.0 instead?

Reply:
`library` is a (custom) compileOnly dependency (not a transitive dependency), the idea here is to compile/test against the earliest version that we claim to support, and then testLatestDep is used to tests against the latest version that we claim to support

Reply:
Thanks for the clarification! To be honest I had a hard time finding the definition of what a `library` dependency was. I downgraded to 1.0.0.